packages/sdk-bundles-api/src/replay/bidirectional/replayable-event.ts

```diff
@@ -0,0 +1,19 @@
+export interface ReplayableEvent {
```

Comment:
Note: we can improve these types over time / document them better. But for now just want to get started on having some shared types / getting things more typesafe.